styles/styles.css

```diff
@@ -1201,6 +1201,56 @@ main.blue-contract .section.section-with-title p {
   width: 1em;
 }
 
+/* Headings classes styles */
+.redesign-v2 .h2 {
+  font-family: var(--ff-volvo-novum);
+
+  /* font-size: 56px; */
+  font-size: var(--f-heading-2-font-size);
+  font-style: normal;
+  font-weight: 300;
+  line-height: 110%;
+  letter-spacing: -0.5px;
+  font-feature-settings: 'clig' off, 'liga' off;
+}
+
+.redesign-v2 h4 {
+  font-family: var(--ff-volvo-novum-medium);
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 125%;
+  letter-spacing: 0.25px;
+  font-feature-settings: 'clig' off, 'liga' off;
+}
+
+.redesign-v2 .h5 {
+  font-family: var(--ff-volvo-novum);
+  font-size: var(--f-heading-5-font-size);
+  font-style: normal;
+  line-height: 125%;
+  font-feature-settings: 'clig' off, 'liga' off;
+}
+
+.redesign-v2 .h6 {
+  font-family: var(--ff-volvo-novum-medium);
+  font-size: var(--f-heading-6-font-size);
+  font-style: normal;
+  line-height: 125%;
+  letter-spacing: 0.15px;
+  font-feature-settings: 'clig' off, 'liga' off;
+
+}
+
+.redesign-v2 .sub-2 {
```

Comment:
What's sub-2? Can we rename it to `subtitle-2`? If we want to have generic class names, they need to be clear.
And if it's a generic style, why we are not using the CSS variables for the font size / letter spacing / line-hegiht? in this case `--f-subtitle-2-font-size`

Reply:
Not problem, changed to `subtitle-2`